l10n_it_corrispettivi/models/account.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Lorenzo Battistini - Agile Business Group
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    @api.model
+    def _default_account(self):
+        if not self._context.get('default_corrispettivi', False):
+            # If this is not a corrispettivo, do nothing
+            return False
+        if self.corrispettivo:
+            return self.env.ref('base.public_user') \
+                .partner_id.property_account_receivable_id.id
+
+    @api.model
+    def _default_partner_id(self):
+        if not self._context.get('default_corrispettivi', False):
+            # If this is not a corrispettivo, do nothing
+            return False
+        return self.env.ref('base.public_user').partner_id.id
+
+    # set default option on inherited field
+    corrispettivo = fields.Boolean(
+        string='Corrispettivo', related="journal_id.corrispettivi",
+        readonly=True, store=True)
+    account_id = fields.Many2one(default=_default_account)
```

Comment:
@SimoRubi questa modifica non è superflua, visto che il conto in fattura viene già preso dal partner selezionato?

Reply:
Vero, rimosso il default